src/main/java/com/epam/test_generator/dto/CaseDTO.java

```diff
@@ -27,8 +28,10 @@
     @Valid
     private List<StepDTO> steps;
 
+    @JsonFormat(pattern = "yyyy-MM-dd@HH:mm:ss")
```

Comment:
Looks like you resolved conflicts improperly while merging from develop. This annotation should not exist in develop and you trying to reintroduce it back again.

Since you fail to merge, my piece of advice is to create new branch and rewrite all code from scratch.